ORB_SLAM3:
    VocabularyPath:  "/home/uas/a_dependencies/catkin_FASTLIOori/src/FAST_LIO/ORB_SLAM3/Vocabulary/ORBvoc.txt"
    SettingsPath:  "/home/uas/a_dependencies/catkin_FASTLIOori_MA/src/FAST_LIO/ORB_SLAM3/Examples/Stereo-Inertial/3001.yaml"
    ImuStereo:
        DoRectify: "false"